src/components/About/about.css

```diff
@@ -0,0 +1,106 @@
+.about-wrapper {
+  width: 340px;
+  margin: 0 auto;
+  border-radius: 20px;
+  background-color: var(--primary-element-color);
+}
+
+.about-title {
+  width: 300px;
+  display: block;
+  margin: 0 auto;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  color: var(--primary-text-color);
+}
+
+.about-text {
+  width: 300px;
+  height: auto;
+  margin: 0 auto;
+  color: var(--secondary-text-color);
+}
+
+.about-text a {
+  word-wrap: break-word;
+  transition: none;
+  color: var(--accent-text-color);
+}
+
+[data-theme='light'] {
+  .about-wrapper {
+    background-color: var(--primary-element-color);
+  }
+}
+
+[data-theme='light'] {
+  .about-title {
+    color: var(--primary-text-color);
+  }
+}
+
+[data-theme='light'] {
+  .about-text {
+    color: var(--secondary-text-color);
+  }
+}
+
+[data-theme='light'] {
+  .about-text a {
+    color: var(--accent-text-color);
+  }
+}
+
+@media (min-width: 769px) {
+  .about-wrapper {
+    width: 1320px;
+    margin: 0 auto;
+    border-radius: 20px;
+    background-color: var(--primary-element-color);
+  }
+
+  .about-title {
+    width: 1240px;
+    display: block;
+    margin: 0 auto;
+    padding-top: 40px;
+    padding-bottom: 40px;
+    color: var(--primary-text-color);
+  }
+
+  .about-text {
+    width: 1240px;
+    height: 590px;
+    margin: 0 auto;
+    color: var(--secondary-text-color);
+  }
+
+  .about-text a {
+    transition: none;
```

Comment:
so many repititions. and I don't know what for

Reply:
done